registration-processor/stage-groups/group-5/pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>io.mosip.registrationprocessor</groupId>
+		<artifactId>stage-groups</artifactId>
+		<version>1.2.0-SNAPSHOT</version>
+	</parent>
+	
+	<artifactId>group-5</artifactId>
+    <version>1.2.0-SNAPSHOT</version>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<dependencies>
+	
+		<!-- Stage Executor -->
+		<dependency>
+			<groupId>io.mosip.registrationprocessor</groupId>
+			<artifactId>mosip-stage-executor</artifactId>
+			<version>1.2.0-SNAPSHOT</version>
+		</dependency>
+		
+		<!-- Stages -->
+		<dependency>
```

Comment:
We should remove external stage from the group since it will be removed later

Reply:
Done.